app.js

```diff
@@ -1,4 +1,51 @@
 //  in this file create an express application - use the middle-ware built into express
 //  to serve up static files from the public directory (index.html and client.js - you 
 //  can also serve up css files from the public directory this way if you want)
-//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
\ No newline at end of file
+//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
+'use strict';
+var express = require('express');
+var trucks = require('./trucks');
+var app = express();
+app.set('json spaces', 4);
+
+app.use(express.static('public'));
+
+// the param method will map a placeholder variable to a callback
+// app.param('name', function (request, response, next) {
+//     var truckName = request.params.name;
+//     var foodTruck = truckName[0].toUpperCase() + truckName.slice(1).toLowerCase();
+//     request.foodTruck = foodTruck;
+//     next();
+// });
+
+app.get('/', function (request, response){
```

Comment:
Because there is an index.html file present, the root route is already handled by the time this route handler is reached.